test/schema-type-test.js

```diff
@@ -3,6 +3,7 @@ import typeBundle from '../fixtures/custom-type-bundle';
 import schemaForType from '../src/schema-for-type';
 import Query from '../src/query';
 import Mutation from '../src/mutation';
+import realTypeBundle from '../fixtures/types'; // eslint-disable-line import/no-unresolved
```

Comment:
I'm unsure about the naming here. It may be better to call `typeBundle `customTypeBundle` and this one `typeBundle` or something. `realTypeBundle` just raises all sorts of questions about what the other bundle is.

Reply:
You're completely right. I will rename this.